packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -210,7 +237,11 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
                     <Text style={styles.restrictedText}>{t('restrictedRegion')}</Text>
                   )}
                 </View>
-                <LinkArrow />
+                {provider.isFeeDataLoading ? (
```

Comment:
We should also check with the designer on what to do in case of an error state. What happens if the quote query fails?

Reply:
This was kind of a placeholder but we decided to disable options if/when the quotes fail as that likely means the provider is unavailable